model/event/subscriptions.go

```diff
@@ -40,25 +41,27 @@ const (
 )
 
 type Subscription struct {
-	ID             bson.ObjectId `bson:"_id"`
-	Type           string        `bson:"type"`
-	Trigger        string        `bson:"trigger"`
-	Selectors      []Selector    `bson:"selectors,omitempty"`
-	RegexSelectors []Selector    `bson:"regex_selectors,omitempty"`
-	Subscriber     Subscriber    `bson:"subscriber"`
-	Owner          string        `bson:"owner"`
-	OwnerType      OwnerType     `bson:"owner_type"`
+	ID             bson.ObjectId     `bson:"_id"`
+	Type           string            `bson:"type"`
+	Trigger        string            `bson:"trigger"`
+	Selectors      []Selector        `bson:"selectors,omitempty"`
+	RegexSelectors []Selector        `bson:"regex_selectors,omitempty"`
+	Subscriber     Subscriber        `bson:"subscriber"`
+	Owner          string            `bson:"owner"`
+	OwnerType      OwnerType         `bson:"owner_type"`
```

Comment:
This is like the tiniest of things, but it's been bugging me for a while: can you move OwnerType to be just above Owner please?